BackEnd/src/controllers/ProjectController.php

```diff
@@ -0,0 +1,234 @@
+<?php
+namespace HostMyDocs\Controllers;
+
+use HostMyDocs\Models\Language;
+use HostMyDocs\Models\Project;
+use HostMyDocs\Models\Version;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Finder\SplFileInfo;
+
+abstract class ProjectController {
```

Comment:
Should not be abstract and should not contains only static methods.
You should create an instance and use the dependency injector of slim